modules/tensor_mechanics/doc/content/source/auxkernels/RankTwoAux.md

```diff
@@ -37,13 +37,16 @@ AuxVariable is used as the argument for the `variable` input parameter in the `R
 
 !alert note title=Elemental vs Nodal Visualization of Quadrature Field Values
 Using an AuxVariable with `family = MONOMIAL` and `order = CONSTANT` will give a constant value of
-the AuxVariable for the entire element, which is computed by taking a volume-weighted average of the
-integration point quantities. Using an AuxVariable with `family = MONOMIAL` and `order = FIRST` or
-higher will result in fields that vary linearly (or with higher order) within each element, which are
-computed using a local least-squares procedure. Because the Exodus mesh format does not support
-higher-order elemental variables, these AuxVariables are output by libMesh as nodal variables for
-visualization purposes. Using higher order monomial variables in this way can produce smoother
-visualizations of results for a properly converged simulation.
+the AuxVariable for the entire element, which is computed by taking a local $L_2$ projection. Using an
+AuxVariable with `family = MONOMIAL` and `order = FIRST` or higher will result in fields that vary linearly
+(or with higher order) within each element. Because the Exodus mesh format does not support higher-order
+elemental variables, these AuxVariables are output by libMesh as nodal variables for visualization purposes.
+Using higher order monomial variables in this way can produce smoother visualizations of results for a properly
+converged simulation.
+
+Using an AuxVariable with `family = LAGRANGE` will give a smooth nodal field of the material property,
+constructed using nodal patch recovery. `patch_polynomial_order` is set to equal the order of the AuxVariable
+by default.
```

Comment:
I think I would re-arrange the above text a little to convey the fact that there are really 3 main options: constant monomial, higher order monomial, and patch recovery for nodal variables. Emphasize that for the best (smoothest, most accurate) results, use the last option, but there is some additional computational cost.